README.md

```diff
@@ -1,94 +1,14 @@
-# Project React Native App 📱
+Description:
 
-In this week's project, you'll be building your first mobile application with React Native!
+React native app that shows pictures of cute dogs when you push the screen,
 
-#### It's time for your creative juices to flow like the Niagara Falls!
+What did I do? 
 
-![Niagara Falls](https://images.unsplash.com/photo-1489447068241-b3490214e879?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80)
+I used expo/vs code and fetched a dog-API onto React Native to create a mobile app. I also used Styled Components for the styling.
 
-This week's project is a very free one. Your only restrictions are that it should be a mobile app created with React Native and using styled components. Also, the app it self should either be:
-  - using one of the sensors of your mobile phone such as the camera, accelerometer or voice assistance etc. 
-  - using an external API to fetch something to your app and then interact with it.
-  - a multiscreen app by using [React Navigation](https://facebook.github.io/react-native/docs/navigation).
-  - a useful app such as a calculator, a stopwatch or a compass. 
+If I had more time I wanted to fetch and diplay each dogs temperament.
```

Comment:
You can do this with adding this line:
`<Text>{photo.breeds[0].temperament}</Text>`
Similar to
`<Text>{photo.breeds[0].name}</Text>`